source/js/main.js

```diff
@@ -0,0 +1,299 @@
+import { storage } from "./storage.js";
```

Comment:
Codacy has a fix for the issue: [There should be no space after '{'.](https://app.codacy.com/gh/cse110-fa21-group27/cse110-fa21-group27/pullRequest?prid=8447291)
```suggestion
import {storage } from "./storage.js";
```